xchange-btcmarkets/src/main/java/org/knowm/xchange/btcmarkets/dto/BTCMarketsBaseResponse.java

```diff
@@ -6,9 +6,9 @@
 
 public class BTCMarketsBaseResponse {
 
-  private final Boolean success;
-  private final String errorMessage;
-  private final Integer errorCode;
+  public final Boolean success;
+  public final String errorMessage;
+  public final Integer errorCode;
```

Comment:
restore the getters in all DTOs

by hand or lombok pls

Reply:
Most PR's ive done and seen gone through have opted for public finals instead of getters ?

Reply:
most DTOs have getters, core DTOs have getters with one exception which will be corrected in a future release

its best t ostick with one convetion

Reply:
and in the case of core DTO we should definitely stick to getters as without them it will be hard to refine the API in a backward compatible manner

Reply:
DTO members should be private final and have getters or if using Lombok they can be droppped.

Reply:
updated in last commits.